.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - '6'
+
+dist: trusty
+sudo: required
+
+env:
+  global:
+  - NODE_ENV=test
+  - LABS_AUTH_SERVICE_pgdb_password=''
+
+addons:
+  postgresql: "9.5"
```

Comment:
@paolochiodi  There is a simple possibility to test against multiple Postgres versions: add in the Travis file a `docker pull` command and start Postgres as docker. This can be automated with the matrix feature that Travis provides. I tried this trick a few days ago and hit some issues starting the Docker but it is probably doable.